Src/Particle/AMReX_TracerParticle_mod_K.H

```diff
@@ -1,208 +1,139 @@
 #ifndef AMREX_TRACERPARTICLE_MOD_K_H
 #define AMREX_TRACERPARTICLE_MOD_K_H
-#include <AMReX_Config.H>
 
+#include <AMReX_Config.H>
 #include <AMReX_FArrayBox.H>
 #include <AMReX_Box.H>
 #include <AMReX_Gpu.H>
 #include <AMReX_Geometry.H>
 #include <AMReX_REAL.H>
 #include <AMReX_IntVect.H>
 #include <AMReX_TracerParticles.H>
-
 #include <cmath>
 
-namespace amrex{
+namespace amrex {
 
+//
+// cic_interpolate computes the velocity at the particle by interpolating from velocity
```

Comment:
it could be any co-located cell-centered field. not just velocity?
should we change uarr to mfarr?